demonstrations/tutorial_kak_theorem.py

```diff
@@ -4,24 +4,25 @@
 The KAK theorem is a beautiful mathematical result from Lie theory, with
 particular relevance for quantum computing. It can be seen as a
 generalization of the singular value decomposition, as it decomposes a group
```

Comment:
```suggestion
generalization of the singular value decomposition (SVD), as it decomposes a group
```
To make it surface better in searches.